### DISCUSSION TITLE
Phoenix-Takımı-başvuru-(Merve Damgacı)

### DISCUSSION THREAD

**Discussion by mervedam** [sonsuzus/BilsemPython#68] [Genel]
Takıma katılmak isterseniz grubunuzu gününüzü ve belirtmek istediğiniz birşey varsa yazabilirsiniz 

-Öyg1 çarşamba Merve damgacı

**Comment by AcelyaP** [comment]
Merhabalar ben Açelya Pirtini ÖYG 1im cumartesi günleri 3 ve 4 derslere giriyorum bu gruba katılmak istiyorum

**Reply by sonsuzus** [reply]
Açelya sen kaptansın :) takım kurman gerekiyor

**Reply by AcelyaP** [reply]
Tamam hocam görmemişim

**Comment by ElifNurSoyler** [comment]
Merhaba ben Elif. Çarşamba 3. ve 4. derslere giriyorum. Gruba katılabilir miyim??

**Reply by mervedam** [reply]
Takıma alındın elif yazdığın için teşekkürler 🙂

**Reply by ElifNurSoyler** [reply]
Ricalar

**Reply by mervedam** [reply]
Elif phoenix takımına kanal açtım telegramdan katıla bilirsin  https://t.me/+3pQP1FvSzIljMGNk
